### DISCUSSION TITLE
Penyajian Grafik

### DISCUSSION THREAD

**Discussion by joejnt** [teamalgoritma/community#51] [Q&A] [answered]
Saya sudah membuat data agregasi dengan code ini:

Total_trx <- aggregate(formula = Total_Trx~Branch_Type, data = dataset, FUN=sum)
Total_trx[order(Total_trx$Total_Trx, decreasing = T),]

Bagaimana cara saya menyajikannya dalam bentuk plot ataupun grafik ya ?

**Comment by tomytjandra** [comment, marked as answer]
Halo Pak @joejnt, berikut adalah cara untuk memvisualisasikan sebuah dataframe menjadi plot menggunakan fungsi pada Base R.

Pertama, saya membuat dataframe sesuai kebutuhan Pak Joe dengan code berikut:
```
total_trx <- data.frame(
  Branch_Type = c("Cabang", "KCP Type B", "KCP Type A"),
  Total_Trx = c(40213213, 2621197, 872184)
)
```

Lalu berdasarkan dataframe tersebut, kita bisa membuat sebuah **barplot untuk melakukan perbandingan** nilai numerik (Total_Trx) untuk setiap data kategoriknya (Branch_Type). Referensi: https://r-graph-gallery.com/208-basic-barplot.html

Berikut cara membuat barplot dengan fungsi Base R:

```
options(scipen = 99) # untuk menghilangkan scientific notation pada plot
barplot(height=total_trx$Total_Trx, names=total_trx$Branch_Type)
```

Penjelasan parameter pada fungsi `barplot()`:
- `height` adalah kolom numerik
- `names` adalah kolom kategorik

Output:
![image](https://user-images.githubusercontent.com/48079239/163766744-c0cf485e-a0a8-428c-af76-404a1bc9f233.png)
